webapp/src/client/app/modules/web-client/vnc/web-client-vnc.component.html

```diff
@@ -67,6 +67,6 @@
     </div>
   </div>
 
-  <iron-remote-gui-vnc #ironGuiElementVnc debugwasm="INFO" verbose="true" scale="fit" flexcenter="true"></iron-remote-gui-vnc>
+  <iron-remote-gui-vnc #ironGuiElementVnc scale="fit" flexcenter="true"></iron-remote-gui-vnc>
```

Comment:
Good enough for now, but I would like an option to enable the logging back in the UI in the future. It’s useful when troubleshooting problems deeper in our implementations.